src/WebJobs.Script/Extensions/FunctionMetadataNameComparer.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using FunctionMetadata = Microsoft.Azure.WebJobs.Script.Description.FunctionMetadata;
+
+namespace Microsoft.Azure.WebJobs.Script.Extensions
+{
+    public class FunctionMetadataNameComparer : IComparer<FunctionMetadata>
```

Comment:
Recommend you just remove this class and simply sort via `functionMetadataList.OrderBy(p => p.Name).ToImmutableArray();`